mllib/src/main/scala/org/apache/spark/mllib/tree/loss/Loss.scala

```diff
@@ -49,4 +49,14 @@ trait Loss extends Serializable {
    */
   def computeError(model: TreeEnsembleModel, data: RDD[LabeledPoint]): Double
```

Comment:
Provide default implementation using other computeError, and then remove overridden copies from child classes